common/windows/build.sh

```diff
@@ -18,7 +18,7 @@ builder_describe "Keyman common Windows modules" \
 
 builder_parse "$@"
 
-if [[ $BUILDER_OS != windows ]]; then
+if [[ $BUILDER_OS != win ]]; then
```

Comment:
This script isn't actually used at this point, but I noticed this when testing. Just a silly.